src/content/registry/shared.js

```diff
@@ -41,7 +45,7 @@ export const createParseCommand =
           continue;
         }
 
-        if (word.startsWith('-')) {
+        if (word.startsWith('-') || isNumber(word)) {
           counterIndex += handleArgument(word, argsAndPackagesWords);
           continue;
         }
```

Comment:
I wrote a detaild comment but it was cleared 😣

This function in the `shared.js` file is a generic function for all the package managers, and it parses a generic CLI line.
This exact `if` identifies if it is an argument part. How each package manager is handling an argument, is it just ignore it, maybe ignore it and its value, maybe extracting an important value from it?

All those are implemented per-package manager and injected as `handleArgument` function.

Currently the `handleArgument` for `yarn` (npm) is very basic, just [moves the index to the next word](https://github.com/yoavsbg/overlay/blob/f1e8ff83d17436ce2a1c02e4434cfa7b42fa7811/src/content/registry/npm.js#L26):
```javascript
(word) => word.length + 1,
```

But now you required to improve this function because we know that if the argument is `--network-timeout`, we expect for a value after the argument, which also must be ignored.

You can see how it implemented with `pip` (Python) where there are a lot of arguments that accepts value:
https://github.com/yoavsbg/overlay/blob/f1e8ff83d17436ce2a1c02e4434cfa7b42fa7811/src/content/registry/python.js#L87

Reply:
fixed.